src/EditorFeatures/CSharpTest/AutomaticCompletion/AutomaticLineEnderTests.cs

```diff
@@ -703,6 +702,144 @@ public void NotAfterAutoPropertyDeclaration()
 }");
         }
 
+        [WorkItem(150480, "devdiv.visualstudio.com")]
```

Comment:
I thought I've seen it somewhere? Hmm.. Anyway, with the myriad of bug databases to go about, I thought workitem number is useful only if its qualified with its database name :smile: